raiden/exceptions.py

```diff
@@ -190,6 +190,10 @@ class ReplacementTransactionUnderpriced(RaidenError):
     """Raised when a replacement transaction is rejected by the blockchain"""
 
 
+class TransactionAlreadyPending(RaidenError):
```

Comment:
please, make it an unrecoverable error, this may change in the future, but for now this is considered bad usage (we assume Raiden controls the account)